modules/roles_profiles/manifests/profiles/notarization_users.pp

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class roles_profiles::profiles::notarization_users {
+
+    case $::operatingsystem {
+        'Darwin': {
+            # Make sure the users profile is required
+            # That is where the user virtual resources are generated
+            require roles_profiles::profiles::users
+            # Lookup the group array and realize their user resource
+            $users = lookup('user_groups.notarization', Array, undef, undef)
+            realize(Users::Single_user[$users])
+
```

Comment:
`/usr/sbin/sysadminctl -addUser notary10 -admin -password -` as root worked for me manually on mac-v3-signing2.mdc1. We could possibly add the password to the commandline if needed.